extensions/transmission/src/utils/renderCells.tsx

```diff
@@ -1,6 +1,5 @@
 import { splitEvery } from "ramda";
 import BitField from "bitfield";
-import { renderToString } from "react-dom/server";
```

Comment:
personally I liked the renderToString approach more, less chances of messing up the resulting output. Any reason to change?

Reply:
Yes, it wasn't compatible with the Raycast requirements anymore. Raycast requires cross-platform compatibility, but react-dom isn't fully cross-platform.